rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/controller/library/CreateNewLibraryNatsController.kt

```diff
@@ -28,7 +27,7 @@ class CreateNewLibraryNatsController(private val libraryService: LibraryService)
             createFailureResponse(it)
         }
 
-    private fun createdLibrary(request: CreateNewLibraryRequest): Library {
+    private fun createdLibrary(request: CreateNewLibraryRequest): LibraryOuterClass.Library {
```

Comment:
Can we avoid this ugly `LibraryOuterClass`?

Reply:
would it be okay if i will add prefix "Mongo" to all my entity in rest module? Then it would be easier to determine **MongoLibrary** from rest module and **Library** which is class generated by protobuf

Reply:
| REST/NATS layer   | Service layer | Repository layer |
|-------------------|--------------|------------------|
| Library*Request, Library, ApiLibrary   | Library, Library*Request      | MongoLibrary    |

Prefix `Mongo` use for mongo model objects.

You can use `Api` prefix for `Library` in request/response